### DISCUSSION TITLE
Will Bootstrap 5 Drop jQuery?

### DISCUSSION THREAD

**Discussion by trymeouteh** [twbs/bootstrap#30834] [General] [answered]
Will Bootstrap 5 Drop jQuery and therefore make Bootstrap have no dependancies, and allow us to use Bootstrap with plain old JavaScript?

**Comment by lorvent** [comment]
well, it still has dependencies

and they mentioned it doesn't need jquery
but if jquery is already included then it works well with jquery too i.e. you can use either methods using plain javascript or using jquery (so whatever worked in v4 will work with v5 too i believe)

**Comment by mdo** [comment, marked as answer]
v5 is dropping jQuery. We still have a Popper.js dependency, though, and would like to update Popper.js to v2.x for our v5 release.

**Reply by trymeouteh** [reply]
Are there plans to remove Popper.js and make Bootstrap have no dependancies?

**Reply by techboyg5** [reply]
> Are there plans to remove Popper.js and make Bootstrap have no dependancies?

No, but you can use the Bootstrap JS bundle which bundles Popper and Bootstrap JS.

**Comment by Johann-S** [comment]
what @lorvent said 👍

**Reply by rohit2sharma95** [reply]
@Johann-S  Why bootstrap is not dropping the support of jQuery permanently?

**Reply by Johann-S** [reply]
To allow people to easily switch from v4 to v5 and in v6 you won't be able to use jQuery at all
